news/templates/news/list.html

```diff
@@ -76,4 +76,30 @@
 		{% translate "ADD_NEWS_ITEM" %}
 	</a>
 </p>
+
+<script>
+window.addEventListener('DOMContentLoaded', function() {
+	var removeBtns = document.querySelectorAll("[id^='remove-btn']");
+
+	for (var i = 0; i < removeBtns.length; i++) {
+		removeBtns[i].addEventListener('click', function(event) {
+			event.preventDefault();
+
+			var currentUrl = window.location.href;
+			var newsId = this.getAttribute('value');
+
+			fetch(currentUrl + newsId + '/remove/', {
+				method: 'POST',
+				headers: {
+					'X-CSRFToken': '{{ csrf_token }}'
```

Comment:
Could you add a comma after `'X-CSRFToken': '{{ csrf_token }}'`?

Reply:
Done